chat-summary/tool.gpt

```diff
@@ -1,15 +1,16 @@
-name: Agent Chat History Summarizer
-description: Summarizes the previous chat conversations when entering a new agent and summarizes the agent conversion when finishing the chat
-context: sys.chat.history
-share output filter: chatFinishSummary
+Name: Agent Chat History Summarizer
+Description: Summarizes the previous chat conversations when entering a new agent and summarizes the agent conversion when finishing the chat
+Context: sys.chat.history
+Share Output Filter: chatFinishSummary
+Type: context
 
 #!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/history.py
 
 ---
-name: chatFinishSummary
-context: sys.chat.current
-params: output: the message
-params: continuation: if the the conversation is still in progress
-params: chat: if this is a chat conversation
+Name: chatFinishSummary
+Context: sys.chat.current
+Params: output: the message
+Params: continuation: if the the conversation is still in progress
+Params: chat: if this is a chat conversation
```

Comment:
I didn't add it to this tool because it didn't look like a context tool to me. It is only shared from a context tool.

Reply:
I think the top level tool in this file is the ONLY context tool in this dir. it is used here: https://github.com/gptscript-ai/context/blob/d90368bcfe2e5c6f2c5a136c454605ad9c7b9ece/multi-agent/tool.gpt#L4

i think summarize.gpt and output.gpt are used like this:
https://github.com/gptscript-ai/context/blob/d90368bcfe2e5c6f2c5a136c454605ad9c7b9ece/chat-summary/output.py#L10
https://github.com/gptscript-ai/context/blob/d90368bcfe2e5c6f2c5a136c454605ad9c7b9ece/chat-summary/history.py#L8
and i think that usage is not as a context tool